### PR TITLE
Update back constants' Ceph config values for 4.7 and below clusters

### LINKED ISSUE #4682: [OCS 4.7] test_validate_ceph_config_values_in_rook_config_override fails because of PR 4622 which added new Ceph config values which are applicable to 4.8 and above

### PR DESCRIPTION
**Pull request by ebenahar**
Fixes #4682

Signed-off-by: Elad Ben Aharon <ebenahar@redhat.com>

### CODE REVIEW

**Review by vavuthu — APPROVED** [review]

File: ocs_ci/ocs/constants.py [hunk 1]
```diff
@@ -1333,6 +1333,16 @@
 
 ROOK_CEPH_CONFIG_VALUES = """
 [global]
+mon_osd_full_ratio = .85
```
Comment (petr-balogh) [inline ocs_ci/ocs/constants.py:1336]: @ebenahar was this change introduced somewhere in the middle of the development of 4.8?  The thing is that we have release-4.7 branch, and if we will start to using it, I am curious if we have those values tweaked already in release-4.7 branch or we need to backport and cherrypick those changes to the release-4.8 and release-4.7 branches?

**Review by petr-balogh — APPROVED** [review]

File: ocs_ci/ocs/constants.py (hunk 1 above)
Comment (ebenahar) [reply ocs_ci/ocs/constants.py:1336]: It was merged in master on July 28 - https://github.com/red-hat-storage/ocs-ci/pull/4623

**Review by AviadP — APPROVED** [review]

File: ocs_ci/ocs/constants.py (hunk 1 above)
Comment (petr-balogh) [reply ocs_ci/ocs/constants.py:1336]: so release 4.7 branch should be still fine

**Comment by ebenahar** [comment]
@petr-balogh do you suggest backporting this to release-4.7 branch?

**Pull request merged**